pkg/model/components/clusterautoscaler.go

```diff
@@ -93,6 +93,9 @@ func (b *ClusterAutoscalerOptionsBuilder) BuildOptions(o interface{}) error {
 	if cas.MaxNodeProvisionTime == "" {
 		cas.MaxNodeProvisionTime = "15m0s"
 	}
+	if cas.CreatePriorityExpenderConfig == nil {
```

Comment:
I recommend changing this one to `true` by default iff the priority expander is set. If not, always leave it unset. That way this won't cause an unnecessary diff for everyone not using the priority expander.